tests/integrationv2/tox.ini

```diff
@@ -5,7 +5,7 @@ skipsdist = True
 [testenv]
 # install pytest in the virtualenv where commands will be executed
 setenv = S2N_INTEG_TEST = 1
-passenv = DYLD_LIBRARY_PATH, LD_LIBRARY_PATH, OQS_OPENSSL_1_1_1_INSTALL_DIR, HOME, TOX_TEST_NAME
+passenv = DYLD_LIBRARY_PATH, LD_LIBRARY_PATH, OQS_OPENSSL_1_1_1_INSTALL_DIR, OPENSSL_1_0_2_INSTALL_DIR, HOME, TOX_TEST_NAME
```

Comment:
The equivalent in nix will be an addition to the [shellHook](https://github.com/aws/s2n-tls/blob/8ee7a3cb07dbd5020310023ad12e2acb151f1b09/flake.nix#L144) stanza, but let's do this as a future change.